projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -17,6 +17,7 @@ class WPCOM_REST_API_V2_Endpoint_Launchpad extends WP_REST_Controller {
 	 * Class constructor
 	 */
 	public function __construct() {
+		require_once __DIR__ . '/../launchpad/launchpad.php';
```

Comment:
I think we can remove this now. We require launchpad.php at a higher level now

https://github.com/Automattic/jetpack/blob/trunk/projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php#L73-L78

Reply:
I've removed the redundant require statement.

Reply:
After removing this, Jetpack tests started failing saying we're making a called to an undefined function for get_launchpad_checklist_by_checklist_slug(). Adding back.